courses/free-courses-en.md

```diff
@@ -750,7 +750,7 @@
 
 ### .NET
 
```

Comment:
```suggestion
### .NET

> :information_source: See also &#8230; [C#](#c-sharp)

```

It lefts moving record to `C Sharp` section (line 237) and adding below it heading...

```md
### C Sharp

> :information_source: See also &#8230; [.NET](#net)


````

Empty lines of above template are important

Reply:
@MahmoudHamdy00 not resolved yet

Crosslinks goes before lists (see for example Angular and Angular JS)

Reply:
Resolved by commit: [`b106347` (#6673)](https://github.com/EbookFoundation/free-programming-books/pull/6673/commits/b106347269925bfe72e559be653461e20b4ef1b8)